src/elements/alert/alert/readme.md

```diff
@@ -69,7 +69,7 @@ As a base rule, opening animations should be active if an alert arrives after th
 | `animation`    | `animation`     | public  | `'open' \| 'close' \| 'all' \| 'none'` | `'all'`            | The enabled animations.                                                                                                                                          |
 | `iconName`     | `icon-name`     | public  | `string`                               | `'info'`           | Name of the icon which will be forward to the nested `sbb-icon`. Choose the icons from https://icons.app.sbb.ch. Styling is optimized for icons of type HIM-CUS. |
 | `isOpen`       | -               | public  | `boolean`                              |                    | Whether the element is open.                                                                                                                                     |
-| `readonly`     | `readonly`      | public  | `boolean`                              | `false`            | Whether the alert is readonly. In readonly mode, there is no dismiss button offered to the user.                                                                 |
+| `readOnly`     | `readOnly`      | public  | `boolean`                              |                    | Whether the alert is readonly. In readonly mode, there is no dismiss button offered to the user.                                                                 |
```

Comment:
Probably not solvable, but the attribute should be readonly (lower case)

Reply:
I think I can add `attribute: 'readonly'` in the property, just for docs generation